misc/extension_api_validation/4.2-stable.expected

```diff
@@ -372,3 +372,10 @@ GH-91382
 Validate extension JSON: Error: Field 'classes/AudioStreamPlaybackPolyphonic/methods/play_stream/arguments': size changed value in new API, from 4 to 6.
 
 Optional arguments added. Compatibility methods registered.
+
+
+GH-93605
+--------
+Validate extension JSON: JSON file: Field was added in a way that breaks compatibility 'classes/Semaphore/methods/post': arguments
```

Comment:
This isn't the line to add, I think it should be:
```suggestion
Validate extension JSON: Error: Field 'classes/Semaphore/methods/post/arguments': size changed value in new API, from 0 to 1.
```
But we can see when the CI is done

Reply:
My bad was the correct one, just hadnt seen before